docs/roadmap.rst

```diff
@@ -136,7 +119,7 @@ Features
 -   better discovery of processes (recommended processes, featured
     processes)
 -   search function for comments and contents
--   project managers can edit permissions in all processes
+-   permissions can be added in all processes
```

Comment:
I think two things are misleading about the new wording:
- for LIQD developers, it has been possible to edit permissions all the time. The sentence suggests the contrary.
- permissions can't just be added, they can also be removed, modified etc.